core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.spec.ts

```diff
@@ -91,7 +93,52 @@ describe('DotContainerContentletService', () => {
         httpMock.expectOne(`v1/containers/form/2?containerId=1`);
     });
 
+    it('should do a request for get the contentlet html code in a specific variant', () => {
+        window.sessionStorage.setItem('variantName', 'Testing');
+
+        const pageContainer: DotPageContainer = {
+            identifier: '1',
+            uuid: '3'
+        };
+
+        const pageContent: DotPageContent = {
+            identifier: '2',
+            inode: '4',
+            type: 'content_type'
+        };
+
+        const dotPage: DotPage = {
+            canEdit: true,
+            canRead: true,
+            canLock: true,
+            identifier: '1',
+            pageURI: '/page_test',
+            shortyLive: 'shortyLive',
+            shortyWorking: 'shortyWorking',
+            workingInode: '2',
+            contentType: undefined,
+            fileAsset: false,
+            friendlyName: '',
+            host: '',
+            inode: '2',
+            name: '',
+            systemHost: false,
+            type: '',
+            uri: '',
+            versionType: ''
+        };
+
+        dotContainerContentletService
+            .getContentletToContainer(pageContainer, pageContent, dotPage)
+            .subscribe();
+        httpMock.expectOne(`v1/containers/content/2?containerId=1&pageInode=2&variantName=Testing`);
+    });
+
     afterEach(() => {
         httpMock.verify();
     });
+
+    afterEach(() => {
+        window.sessionStorage.removeItem('variantName');
```

Comment:
can we add this line in above `afterEach` Cycle and remove this extra  `afterEach`?

Reply:
done https://github.com/dotCMS/core/pull/24159/commits/0a5d748ba5cc237b7e05d6d63b451e9142b5620c#diff-2288713eef0847dd9f6a81ce4c0599266039202d94f3d30a012316ae93856e2eL141